packages/astro/test/core-image-svg.test.js

```diff
@@ -0,0 +1,264 @@
+					assert.equal($svg.attr('height'), '48');
+					assert.equal($svg.attr('width'), '48');
+					assert.equal(!!$svg.attr('size'), false);
+				});
+				it('has height and width overridden - size set', () => {
+					let $svg = $('#override-attrs svg');
+					assert.equal($svg.length, 1);
+					assert.equal($svg.attr('height'), '16');
+					assert.equal($svg.attr('width'), '16');
+					assert.equal(!!$svg.attr('size'), false);
+				});
+			});
+			describe('inline', () => {
+				let $;
+				before(async () => {
+					let res = await fixture.fetch('/inline');
+					let html = await res.text();
+					$ = cheerio.load(html, { xml: true });
+				});
+
+				it('adds the svg into the document directly', () => {
+					let $svg = $('#inline svg');
+					assert.equal($svg.length, 1);
+					assert.equal(!!$svg.attr('viewBox'), true);
+					assert.equal($svg.attr('height'), '1em');
+					assert.equal($svg.attr('width'), '1em');
+					assert.equal($svg.attr('role'), 'img');
+					assert.equal(!!$svg.attr('inline'), false);
+
+					const $symbol = $svg.find('symbol')
+					assert.equal($symbol.length, 0);
+					const $use = $svg.find('use')
+					assert.equal($use.length, 0);
+					const $path = $svg.find('path');
+					assert.equal($path.length, 1);
+				});
+				it('adds the svg into the document and overrides the dimensions', () => {
+					let $svg = $('#inline-with-size svg');
+					assert.equal($svg.length, 1);
+					assert.equal(!!$svg.attr('viewBox'), true);
+					assert.equal($svg.attr('height'), '24');
+					assert.equal($svg.attr('width'), '24');
+					assert.equal($svg.attr('role'), 'img');
+					assert.equal(!!$svg.attr('inline'), false);
+
+					const $symbol = $svg.find('symbol')
+					assert.equal($symbol.length, 0);
+					const $use = $svg.find('use')
+					assert.equal($use.length, 0);
+					const $path = $svg.find('path');
+					assert.equal($path.length, 1);
+				})
+			});
+			describe('title', () => {
+				let $;
+				before(async () => {
+					let res = await fixture.fetch('/title');
+					let html = await res.text();
+					$ = cheerio.load(html, { xml: true });
+				});
+
+				it('adds a title into the SVG', () => {
+					let $svg = $('#base svg');
+					assert.equal($svg.length, 1);
+					assert.equal(!!$svg.attr('title'), false);
+
+					const $title = $('#base svg > title');
+					assert.equal($title.length, 1);
+					assert.equal($title.text(), 'GitHub Logo')
+				});
+			});
+			describe('strip', () => {
+				let $;
+				before(async () => {
+					let res = await fixture.fetch('/strip');
+					let html = await res.text();
+					$ = cheerio.load(html, { xml: true });
+				});
+
+				it('removes unnecessary attributes', () => {
+					let $svg = $('#base svg');
+					assert.equal($svg.length, 1);
+					assert.equal(!!$svg.attr('xmlns'), false);
+					assert.equal(!!$svg.attr('xmlns:xlink'), false);
+					assert.equal(!!$svg.attr('version'), false);
+				});
+			});
+			describe('additional props', () => {
```

Comment:
Would be nice to assert that the default `role="img"` is overwritable. Now that I think of it, we should also double check that `role="img"` is the best default since I know `aria-hidden` and `role="presentation"` are also used sometimes.

Reply:
Will check for some supporting material on this :) Added a test in the mean time to assert it's overridable

Reply:
Everything I'm finding still says `role="img"` for image SVGs paired with a `title` element and/or `aria-label`/`aria-labelledby`. 

For removing it from the accessibility tree, it is still recommended to use `aria-hidden=true`.